### DISCUSSION TITLE
Where can I get the Butterfly Tracing documentation?

### DISCUSSION THREAD

**Discussion by marziah05** [ThreeMammals/Ocelot#1885] [Q&A] [answered]
in tracing section of Ocelot documentation, Ocelot.Tracing.Butterfly package is used. but where can I get the Butterfly documentation?

**Comment by raman-m** [comment, marked as answer]
Hajera,
We've updated Tracing docs with [20.0.1](https://github.com/ThreeMammals/Ocelot/releases/tag/20.0.1) release in October'23.
Updated docs: [Butterfly | Tracing](https://ocelot.readthedocs.io/en/latest/features/tracing.html#butterfly)
- the [Butterfly](https://github.com/liuhaoyang/butterfly) documentation
